portal/guides/on-demand-minting.mdx

```diff
@@ -67,29 +54,63 @@ export default function useProtectedPage() {
 }
 ```
 
-## 2. Create the layout for our page
+## 2. Use the thirdweb and chakra providers
+
+We need to use the thirdweb and chakra providers. You can use the [thirdweb nextjs starter](https://github.com/thirdweb-example/next-typescript-starter) to kickstart this. Here we are using the Mumbai chain.
```

Comment:
we're not explaining 'why' we need these, and what is chakra? a quick one liner explanation here would help.

Reply:
done